server/mergin/sync/permissions.py

```diff
@@ -129,6 +141,20 @@ def check(cls, project, user):
                 )
             )
 
+    @classmethod
+    def get_user_project_role(self, project: Project, user: User) -> Optional[str]:
```

Comment:
How about returning ProjectRole, we would not need to compare strings at other places (e.g. workspace.py)